dev-libs/libgnt/libgnt-2.14.1.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="Text-mode UI toolkit"
+DESCRIPTION="libgnt is a GLib-based ncurses toolkit for creating text-mode \
+graphical user interfaces quickly and easily."
+HOMEPAGE="https://pidgin.im/"
+COPYRIGHT="2006-2021 libgnt contributors"
+LICENSE="GNU GPL v2"
+REVISION=1
+SOURCE_URI="http://downloads.sourceforge.net/project/pidgin/libgnt/$portVersion/libgnt-$portVersion.tar.xz"
+CHECKSUM_SHA256="5ec3e68e18f956e9998d79088b299fa3bca689bcc95c86001bc5da17c1eb4bd8"
+SOURCE_DIR="libgnt-$portVersion"
+PATCHES="libgnt-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+versionCompat="$portVersion compat >= ${portVersion%%.*}"
+
+PROVIDES="
+	libgnt$secondaryArchSuffix = $portVersion
+	lib:libgnt$secondaryArchSuffix = $versionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+	lib:libpanelw$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libgmodule_2.0$secondaryArchSuffix
+	lib:libgthread_2.0$secondaryArchSuffix
+	"
+PROVIDES_devel="
+	libgnt$secondaryArchSuffix = $portVersion
+	devel:libgnt$secondaryArchSuffix = $versionCompat
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	libgnt$secondaryArchSuffix == $portVersion base
+	"
+if [ -z "$secondaryArchSuffix" ]; then
+	ARCHITECTURES_doc="any"
+	PROVIDES_doc="
+		libgnt_doc = $portVersion
+		"
+	REQUIRES_doc=""
+fi
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgmodule_2.0$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	devel:libgthread_2.0$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:gtkdoc_scan
+	cmd:meson
+	cmd:ninja
+	cmd:python2
```

Comment:
As it seems the other OS's don't require it I think we can leave it out (python2.7 is deprecated anyway now, and with a check for python3.7 it says it can't find python2), it doesn't break the build so should be ok.
I think I have something for libpurple ready (will be for later), not finetuned but building, maybe you could check both out with caya later then? (too see for any changes - make backups for you current hpkg packages before doing so) :)